compiler/packages/babel-plugin-react-compiler/src/ReactiveScopes/BuildReactiveBlocks.ts

```diff
@@ -183,6 +183,7 @@ function visitBlock(context: Context, block: ReactiveBlock): void {
         context.append(stmt, stmt.label);
         break;
       }
+      case "pruned-scope":
```

Comment:
```suggestion
```

Unecessary case

Reply:
No, this is necessary. We use exhaustive switch cases with Typescript to make sure we don’t silently fail to handle cases.